README.md

```diff
@@ -11,6 +11,7 @@ The Taxonomy module add the capability to add and edit simple taxonomies within
 ## Requirements
 
  * SilverStripe 4.0+
+ * SilverStripe Admin Module 4.0.2+
```

Comment:
There is no 4.0.2 version of Admin module perhaps you mean 1.0.2 (https://github.com/silverstripe/silverstripe-admin/releases/tag/1.0.2).

If you have a requirement on that version it also needs to be added to `composer.json` (`"silverstripe/admin": "^4.0.2"`)